test/TESTING_GUIDELINES.md

```diff
@@ -1,85 +1,34 @@
 # Testing guidelines
 
-Guide to Cucumber acceptance tests for JS SDK.
-
-**Note:** Currently tests can be run only by Onfido people who have access to `monster` repository on Onfido's Bitbucket.
-
-## Setup
+Guide to UI tests for JS SDK.
 
 ### Software requirements
 
-1. Ruby
-    - https://rvm.io/rvm/install
-    - aim for at least version 2.3.0 (test should also run on older versions but nothing can be guaranteed here)
-    - make sure `gem` is installed alongside with Ruby
-2. Google Chrome (latest version)
-3. Chromedriver (WebDriver for Chrome)
-    - `brew cask install chromedriver`
-    - **Note:** if you have chromedriver already installed, make sure it's the latest version: `brew cask upgrade chromedriver`
-4. Firefox (latest version)
-5. Geckdriver (WebDriver for Firefox)
-    - `brew install geckodriver`
-    - **Note:** if you have geckodriver already installed, make sure it's the latest version: `brew upgrade geckodriver`
-6. Bundler
-    - `gem install bundler`
-7. Other
-    - Make sure you have at least READ access to `monster` repository on Onfido's Bitbucket (it will work only for Onfidoers)
-
-### Test environment
-
-Tests can be run on `test.sh` script runs on two different environments, depending on where it was called:
-- if called by **CI**, it'll run webpack against `ENV=PRODUCTION`
-- if called **locally**, it'll run webpack against `ENV=DEVELOPMENT`
-
-## Running tests locally
-
-**Express run:**
-
-- Run `test.sh` script
-  - it will start webpack server on development environment, install dependencies and execute all `.feature` tests on Chrome browser by default
-  - you can run against other browser than Chrome, just specify BROWSER env variable on test.sh execution, i.e: `BROWSER=firefox bash test.sh`
+1. Google Chrome (latest version)
+2. Chromedriver (WebDriver for Chrome)
+  - `brew cask install chromedriver`
+  - **Note:** if you have chromedriver already installed, make sure it's the latest version: `brew cask upgrade chromedriver`
 
-**Custom run:**
+### Running tests locally
 
-In `test/` directory:
-1. Run `bundle install` to install ruby dependencies
-2. Run `bundle exec cucumber BROWSER=chrome SDK_URL=${SDK_URL} USE_SECRETS=false SEED_PATH=false DEBUG=false`, where you can customize:
-    - `BROWSER` - browser to run tests against (possible choices: `chrome`, `firefox`)
-    - `SDK_URL` - url to deployed SDK (possible choices: any surge link, or `"https://localhost:8080/?async=false"` when webpack started locally using `npm run dev`.
-    **Note:** It's very important to add `?async=false` GET parameter to the end of any URL so that tests wait for JWT to be generated)
+ 1. Run `test.sh` script OR
+ 2. Run: 
+  - `npm run build:dev`
+  - in seprate terminal window run `npm run travis`
```

Comment:
I think `npm run travis` is run by the test js runner, no?

Reply:
True.
Deleted.